internal/pkg/cli/cli.go

```diff
@@ -128,6 +128,15 @@ func indentListItem(multiline string) string {
 	return strings.Join(prefixedLines, "\n")
 }
 
+func indentBy(multiline string, indentCount int) string {
```

Comment:
```suggestion
func indentedBy(multiline string, indentCount int) string {
```

Reply:
I meant for `indent` to be a verb, as in `indent something by` 🤔 If this sounds good to you I will keep it as `indentBy`